engine/engine_test.go

```diff
@@ -150,7 +152,36 @@ func TestTofuEngine_Run(t *testing.T) {
 	assert.Contains(t, output, "Usage: tofu [global options] <subcommand> [args]")
 }
 
+func TestTofuEngineError(t *testing.T) {
+	t.Parallel()
+	engine := &TofuEngine{}
+	mockStream := &MockRunServer{}
+
+	cmd := "tofu"
+	args := []string{"not-a-valid-command"}
+	req := &tgengine.RunRequest{
+		Command: cmd,
+		Args:    args,
+	}
+	err := engine.Run(req, mockStream)
+	assert.NoError(t, err)
```

Comment:
NIT: This should be a require.

I'll cut a PR to have the same linting here that we use for Terragrunt.